### DISCUSSION TITLE
--disable-search-engine-choice-screen

### DISCUSSION THREAD

**Discussion by dornech** [GCuser99/SeleniumVBA#116] [Ideas]
For Chrome it might be an idea to set this argument as a standard.

**Comment by 6DiegoDiego9** [comment]
Yep, I agree.

Try by adding the following line before "Case svbaBrowser.Edge" in the CreateCapabilities function:
caps.AddArguments "--disable-search-engine-choice-screen"

Please let us know if it works.

**Comment by dornech** [comment]
I am used to use Chrome. Should be added there.  :-) 
So far I use 
`objWebCapabilities.AddArguments "--disable-search-engine-choice-screen"`
and it works

**Reply by 6DiegoDiego9** [reply]
before "Case svbaBrowser.Edge" = in the Chrome branch

**Comment by GCuser99** [comment]
What does it do? Are you aware that you can preload any capabilities, per browser, using the `preload_capabilities_file_path` setting in the SeleniumVBA.ini file? See [here for more info](https://github.com/GCuser99/SeleniumVBA/wiki#advanced-customization).

**Reply by 6DiegoDiego9** [reply]
![Screenshot_20241012_224547_com.facebook.katana.jpg](https://github.com/user-attachments/assets/646671bf-534a-4d09-bfbb-67cc4c833984)

It's just this annoying thing, that I believe nobody wants to see at the beginning of an automated session.

**Comment by GCuser99** [comment]
Got it. It's not happening to me in N. America but that would be annoying! I'll make the update soon. Thx.

**Comment by GCuser99** [comment]
@6DiegoDiego9, does it only happen with Chrome? Or does it happen with Edge too?

**Comment by dornech** [comment]
To be honest: working with Chrome, so I do not know.  I think it is a speficic issue due to EU regulation.

But yes, working with a INI file might work as well, so far I have been working with SeleniumBasic for my Selenium VBA stuff so my focus is merely quick 1:1 migration.  My big fear the other day was, SeleniumBasic would not work from one day to the other and so I migrated most of my stuff to Python (with a VBA caller, i. e. GUI in VBA/Office and Selenium logic in Python). This works quite well and I learned Python. However, development in VBA is little bit more flexible (ad hoc code changes) and I like SelenumVBA very much because it is much more complete and much more similar to the Python API of the Selenium binding.

**Comment by GCuser99** [comment]
Fixed in v5.6 - argument to prevent search-engine choice is added in the background if using Chrome.
